earn/src/pages/boost/ManageBoostPage.tsx

```diff
@@ -264,6 +182,18 @@ export default function ManageBoostPage() {
           </div>
         </Container>
       )}
+      {cardInfo && (
+        <BurnBoostModal
+          isOpen={isBurnModalOpen}
+          cardInfo={cardInfo}
+          oracleSeed={oracleSeed}
```

Comment:
Probably want to fetch the oracleSeed inside the modal, otherwise it's quite likely to be stale, no?